### ISSUE TITLE
Compatible with svelte@next?

### ISSUE THREAD

**Issue by janosh** [timhall/svelte-apollo#75]
If I run
```sh
mkdir svelte-next && cd svelte-next && npm init svelte@next && npm i && npm i -D svelte-apollo
```

and then try to import `setClient`

```svelte
<script>
  import Counter from "$components/Counter.svelte";
  import { setClient } from "svelte-apollo";
</script>
```

in `index.svelte`, I get

> 500 Failed to resolve module specifier "process". Relative references must start with either "/", "./", or "../".

Can this be worked around or does `svelte-apollo` need an update to be used with `svelte@next`?

**Comment by georgecrawford** [comment]
I don't really know what I'm doing, but I think this tip has helped: https://github.com/snowpackjs/snowpack/discussions/1477#discussioncomment-122780

**Comment by riez** [comment]
did you able to get this working on `svelte@next`? i'm still trying to findout how to implement graphql client on it..

**Comment by janosh** [comment]
@riez Nope, I ended up using simple `fetch` requests to my GraphQL endpoint instead of an Apollo client. No caching that way, but I'm requesting relatively small amounts of data anyway.

**Comment by riez** [comment]
I see ..  I was thinking to implementing that way too.. but right now I'm trying another package before that..
